docs/core/whats-new/index.md

```diff
@@ -3,15 +3,17 @@ title: What's new in .NET Core
 description: Learn about the new features found in .NET Core.
```

Comment:
Should we even have this article? It's just duplicating the left side of the site with links to the child articles. The intro is misleading as this article doesn't provide a summary at all. The linked articles do. I think the ms.docs design is to move away from these types of articles and let the navigation system on the left handle it.

Reply:
but then what do you if someone types https://docs.microsoft.com/en-us/dotnet/core/whats-new/. You need an index.md file for that to work. 

Other services put all the files on the root or don't have all sub-paths working, which I think it's bad.

Reply:
Yes, I see what you mean. To rectify this, I would remove the href link in the left-side toc so that it makes the node a collapsible version, or have it point to the latest whats new page. This way when someone clicks on it they dont get pointed to the index page and then have to navigate a 2nd time. But the index page is still there for any existing links.

Reply:
We could remove the link from the TOC completely as well and just keep the index for navigation. That could work. But I agree with you that the text is misleading. This can be improved.